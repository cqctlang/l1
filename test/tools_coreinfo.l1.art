args=["coreinfo.cqct","../artifacts/slayer_x86_64_ubuntu_1004.core.gz"];
@include "../tools/coreinfo.cqct"
