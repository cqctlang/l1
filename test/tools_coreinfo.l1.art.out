Note 0 (CORE) is:  64-bit PRSTATUS:
	pr_cursig: 11
	pr_pid:    12075
	pr_ppid:   11019
	pr_pgrp:   12075
	pr_sid:    11019
	reg  0:    0x00000000
	reg  1:    0x00000000
	reg  2:    0x7fff11ce0b60
	reg  3:    0x004005b0
	reg  4:    0x7fff11ce0a80
	reg  5:    0x00000000
	reg  6:    0x00000202
	reg  7:    0x7fff11ce0800
	reg  8:    0x7f3068d8b180
	reg  9:    0x7f3068fd5580
	reg 10:    0x00000000
	reg 11:    0xffffffffffffffff
	reg 12:    0x0000000b
	reg 13:    0x0000000b
	reg 14:    0x00002f2b
	reg 15:    0x0000003e
	reg 16:    0x7f3068c88e07
	reg 17:    0x00000033
	reg 18:    0x00000202
	reg 19:    0x7fff11ce0a58
	reg 20:    0x0000002b
	reg 21:    0x7f30691db700
	reg 22:    0x00000000
	reg 23:    0x00000000
	reg 24:    0x00000000
	reg 25:    0x00000000
	reg 26:    0x00000000
	pr_fpvalid 1
Note 1 (CORE) is:  64-bit PRPSINFO:
	pr_state:  0
	pr_sname:  82
	pr_zomb:   0
	pr_nice:   0
	pr_flag:   4203520
	pr_uid:    1263
	pr_gid:    1020
	pr_pid:    12075
	pr_ppid:   11019
	pr_pgrp:   12075
	pr_sid:    11019
	pr_fname:  slayer
	pr_psargs: slayer 11 
Note 2 (CORE) is:  NT_AUXV (auxiliary vector) of unknown size 304.
Note 3 (CORE) is:  NT_FPREGSET (floating point registers) of unknown size 512.
